### PR TITLE
Fix 3 flaky tests in RdapResponseJsonTest

### PR DESCRIPTION
**Pull request by YiweiLi4**
Test `domain_serialization_test` is flaky since the JSON generator doesn't write the values in a deterministic order. So I configured the mapper and changed the order of values in the expected String.

### CODE REVIEW

**Comment by eshryane** [comment]
Thanks for your contribution @YiweiLi4 we are working on the RDAP service now and we will include this for reviw

**Review by maggarwal13 — APPROVED** [review]

**Comment by eshryane** [comment]
@YiweiLi4 the unit tests in ```RdapResponseJsonTest.java``` are not successful for me, can you check please?

**Comment by YiweiLi4** [comment]
Sorry for the late reply. It is because I modified the helper function `marshal`, which leads to the failure of the unit tests for `nameserver_serialization_test` and `ip_serialization_test`. I just rearranged the order of values in the corresponding strings. It should work now. Let me know if there are still any problems:)

File: whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapResponseJsonTest.java [hunk 1]
```diff
@@ -494,6 +495,7 @@ private JsonFactory createJsonFactory() {
 
         objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         objectMapper.configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, true);
+        objectMapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
```
Comment (petchells) [inline whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapResponseJsonTest.java:498]: This call is deprecated -- replace with...

JsonMapper.builder().configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);

**Comment by petchells** [comment]
Hi, I just tweaked your PR to remove the deprecated calls. For me it works exactly the same as before -- if you can confirm that then I'll merge it. Tnx!
~Paul

**Comment by YiweiLi4** [comment]
[petchells](https://github.com/petchells) Looks good for me. Thank you!

**Comment by petchells** [comment]
merged to fix-unstable-rdap-test so it runs on RIPE's ci/cd

**Pull request closed without merging**